README.md

```diff
@@ -108,6 +108,75 @@ It will essentially run `rebar3 format` inside `test_app`.
 Add modules with any "tricky" formatting you want to `test_app/src`, and push them to github _including_ the `after` results.
 The `after` results can be tought as the **expected output** behaviour.
 
+---
+
+## Proposed Workflow
+When we created this tool, we envisioned a workflow for teams where each member can use their preferred style for code formatting.
+The idea is to take advantage of `rebar3` profiles and write the following on your `rebar.config` file:
+
+```erlang
+%% The canonical format used when pushing code to the central repository
+{format, [
+    {files, ["src/*.erl", "include/*.hrl", "test/*.erl"]},
+    {formatter, default_formatter},
+    {options, #{paper => 100}}
+]}.
+{profiles, [
+    {brujo, [
+        {format, [
+            {files, ["src/*.erl", "include/*.hrl", "test/*.erl"]},
+            {formatter, rok_formatter}, % I prefer comma-first formatting
+            {options, #{paper => 100}}
+        ]}
+    ]},
+    {miriam, [
+        {format, [
+            {files, ["src/*.erl", "include/*.hrl", "test/*.erl"]},
+            {formatter, default_formatter},
+            {options, #{
+                inline_clause_bodies => false, % she doesn't like one-liners
+                inline_items => all % but she doesn't like long lists of items
```

Comment:
"she *does* like", right?

Reply:
Ooops!

Reply:
No, she doesn't like lists of items spanning many lines.